tests/fakeprotocol_smartcamera.py

```diff
@@ -176,6 +181,16 @@ async def _send_request(self, request_dict: dict):
                             return {"error_code": -1}
                     break
             return {"error_code": 0}
+        elif method == "get":
+            it = iter(request_dict)
+            next(islice(it, 1, 1), None)
+            module = next(it)
+            get_method = f"get_{module}"
+            if get_method in info:
+                result = copy.deepcopy(info[get_method]["get"])
+                return {**result, "error_code": 0}
+            else:
+                return {"error_code": -1}
```

Comment:
This is the first test to use a plain `get` method as we're not using them elsewhere yet.